### PR TITLE
Bump Traefik to v1.4.0-rc3

### PR DESCRIPTION
**Pull request by ldez**
Hello,

This PR updates Traefik to v1.4.0-rc3.

/cc @vdemeester @emilevauge

Cheers

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index e753729..4df3fa0 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -6,3 +6,5 @@ traefik:roquefort @ amd64
 traefik:roquefort @ arm32v6
 traefik:roquefort @ arm64v8
 traefik:roquefort-alpine @ amd64
+traefik:roquefort-alpine @ arm32v6
+traefik:roquefort-alpine @ arm64v8
diff --git a/_bashbrew-list b/_bashbrew-list
index 2e418c7..4df7a8b 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -4,8 +4,8 @@ traefik:1.3.8
 traefik:1.3.8-alpine
 traefik:1.4
 traefik:1.4-alpine
-traefik:1.4.0-rc2
-traefik:1.4.0-rc2-alpine
+traefik:1.4.0-rc3
+traefik:1.4.0-rc3-alpine
 traefik:alpine
 traefik:latest
 traefik:raclette
@@ -18,5 +18,5 @@ traefik:v1.3.8
 traefik:v1.3.8-alpine
 traefik:v1.4
 traefik:v1.4-alpine
-traefik:v1.4.0-rc2
-traefik:v1.4.0-rc2-alpine
+traefik:v1.4.0-rc3
+traefik:v1.4.0-rc3-alpine
diff --git a/traefik_roquefort-alpine/Dockerfile b/traefik_roquefort-alpine/Dockerfile
index 5e47d74..47275dd 100644
--- a/traefik_roquefort-alpine/Dockerfile
+++ b/traefik_roquefort-alpine/Dockerfile
@@ -9,7 +9,7 @@ RUN set -ex; \
 		*) echo >&2 "error: unsupported architecture: $apkArch"; exit 1 ;; \
 	esac; \
 	apk add --no-cache --virtual .fetch-deps libressl; \
-	wget -O /usr/local/bin/traefik "https://github.com/containous/traefik/releases/download/v1.4.0-rc2/traefik_linux-$arch"; \
+	wget -O /usr/local/bin/traefik "https://github.com/containous/traefik/releases/download/v1.4.0-rc3/traefik_linux-$arch"; \
 	apk del .fetch-deps; \
 	chmod +x /usr/local/bin/traefik
 COPY entrypoint.sh /
@@ -22,5 +22,5 @@ LABEL org.label-schema.vendor="Containous" \
       org.label-schema.url="https://traefik.io" \
       org.label-schema.name="Traefik" \
       org.label-schema.description="A modern reverse-proxy" \
-      org.label-schema.version="v1.4.0-rc2" \
+      org.label-schema.version="v1.4.0-rc3" \
       org.label-schema.docker.schema-version="1.0"
diff --git a/traefik_roquefort/Dockerfile b/traefik_roquefort/Dockerfile
index bb94523..cd612b7 100644
--- a/traefik_roquefort/Dockerfile
+++ b/traefik_roquefort/Dockerfile
@@ -9,5 +9,5 @@ LABEL org.label-schema.vendor="Containous" \
       org.label-schema.url="https://traefik.io" \
       org.label-schema.name="Traefik" \
       org.label-schema.description="A modern reverse-proxy" \
-      org.label-schema.version="v1.4.0-rc2" \
+      org.label-schema.version="v1.4.0-rc3" \
       org.label-schema.docker.schema-version="1.0"
diff --git a/traefik_roquefort/traefik b/traefik_roquefort/traefik
index 96a9a33..9384399 100755
Binary files a/traefik_roquefort/traefik and b/traefik_roquefort/traefik differ
```

</details>

File: library/traefik [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
+GitCommit: 167c6ac81bbf39bebfbe4fdfef0e931ab33e9cac
 amd64-Directory: scratch/amd64
 arm32v6-Directory: scratch/arm
 arm64v8-Directory: scratch/arm64
 
-Tags: v1.4.0-rc2-alpine, 1.4.0-rc2-alpine, v1.4-alpine, 1.4-alpine, roquefort-alpine
-Architectures: amd64
-GitCommit: 57f83af9d2719e5f25e6c769fd2640f973b519dc
+Tags: v1.4.0-rc3-alpine, 1.4.0-rc3-alpine, v1.4-alpine, 1.4-alpine, roquefort-alpine
+Architectures: amd64, arm64v8, arm32v6
```
Comment (yosifkit) [inline library/traefik]: This one still needs to wait on https://github.com/gliderlabs/docker-alpine/issues/304 for non-amd64 architectures. :cry:

Comment (ldez) [reply library/traefik]: yes sorry.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 2e418c7..4df7a8b 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -4,8 +4,8 @@ traefik:1.3.8
 traefik:1.3.8-alpine
 traefik:1.4
 traefik:1.4-alpine
-traefik:1.4.0-rc2
-traefik:1.4.0-rc2-alpine
+traefik:1.4.0-rc3
+traefik:1.4.0-rc3-alpine
 traefik:alpine
 traefik:latest
 traefik:raclette
@@ -18,5 +18,5 @@ traefik:v1.3.8
 traefik:v1.3.8-alpine
 traefik:v1.4
 traefik:v1.4-alpine
-traefik:v1.4.0-rc2
-traefik:v1.4.0-rc2-alpine
+traefik:v1.4.0-rc3
+traefik:v1.4.0-rc3-alpine
diff --git a/traefik_roquefort-alpine/Dockerfile b/traefik_roquefort-alpine/Dockerfile
index 5e47d74..47275dd 100644
--- a/traefik_roquefort-alpine/Dockerfile
+++ b/traefik_roquefort-alpine/Dockerfile
@@ -9,7 +9,7 @@ RUN set -ex; \
 		*) echo >&2 "error: unsupported architecture: $apkArch"; exit 1 ;; \
 	esac; \
 	apk add --no-cache --virtual .fetch-deps libressl; \
-	wget -O /usr/local/bin/traefik "https://github.com/containous/traefik/releases/download/v1.4.0-rc2/traefik_linux-$arch"; \
+	wget -O /usr/local/bin/traefik "https://github.com/containous/traefik/releases/download/v1.4.0-rc3/traefik_linux-$arch"; \
 	apk del .fetch-deps; \
 	chmod +x /usr/local/bin/traefik
 COPY entrypoint.sh /
@@ -22,5 +22,5 @@ LABEL org.label-schema.vendor="Containous" \
       org.label-schema.url="https://traefik.io" \
       org.label-schema.name="Traefik" \
       org.label-schema.description="A modern reverse-proxy" \
-      org.label-schema.version="v1.4.0-rc2" \
+      org.label-schema.version="v1.4.0-rc3" \
       org.label-schema.docker.schema-version="1.0"
diff --git a/traefik_roquefort/Dockerfile b/traefik_roquefort/Dockerfile
index bb94523..cd612b7 100644
--- a/traefik_roquefort/Dockerfile
+++ b/traefik_roquefort/Dockerfile
@@ -9,5 +9,5 @@ LABEL org.label-schema.vendor="Containous" \
       org.label-schema.url="https://traefik.io" \
       org.label-schema.name="Traefik" \
       org.label-schema.description="A modern reverse-proxy" \
-      org.label-schema.version="v1.4.0-rc2" \
+      org.label-schema.version="v1.4.0-rc3" \
       org.label-schema.docker.schema-version="1.0"
diff --git a/traefik_roquefort/traefik b/traefik_roquefort/traefik
index 96a9a33..9384399 100755
Binary files a/traefik_roquefort/traefik and b/traefik_roquefort/traefik differ
```

</details>

Build test of #3464; 39abeca7 (`traefik`):

```console

$ bashbrew build traefik:v1.4.0-rc3
Using bashbrew/cache:b666dff2878685d6579ad432c1fb7eb19009f6c5e9097a7ee75d68dc9ad1246a (traefik:v1.4.0-rc3)
Tagging traefik:v1.4.0-rc3
Tagging traefik:1.4.0-rc3
Tagging traefik:v1.4
Tagging traefik:1.4
Tagging traefik:roquefort

$ test/run.sh traefik:v1.4.0-rc3
testing traefik:v1.4.0-rc3
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed


$ bashbrew build traefik:v1.4.0-rc3-alpine
Using bashbrew/cache:d2908f7e0f418f0e6e8aee32640fd7ece8678679d7fc37141a3e7e42a88b2a65 (traefik:v1.4.0-rc3-alpine)
Tagging traefik:v1.4.0-rc3-alpine
Tagging traefik:1.4.0-rc3-alpine
Tagging traefik:v1.4-alpine
Tagging traefik:1.4-alpine
Tagging traefik:roquefort-alpine

$ test/run.sh traefik:v1.4.0-rc3-alpine
testing traefik:v1.4.0-rc3-alpine
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...passed


$ bashbrew build traefik:v1.3.8
Using bashbrew/cache:9c9ef67dd208a0925fcdc5d6eff4bf85982cd2ecb74359d7684d5f2ce9b10506 (traefik:v1.3.8)
Tagging traefik:v1.3.8
Tagging traefik:1.3.8
Tagging traefik:v1.3
Tagging traefik:1.3
Tagging traefik:raclette
Tagging traefik:latest

$ test/run.sh traefik:v1.3.8
testing traefik:v1.3.8
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed


$ bashbrew build traefik:v1.3.8-alpine
Using bashbrew/cache:f4a8ba281f86a839b95b28c3f4f6bc012b859d5f2c7cab38b0435b6315149731 (traefik:v1.3.8-alpine)
Tagging traefik:v1.3.8-alpine
Tagging traefik:1.3.8-alpine
Tagging traefik:v1.3-alpine
Tagging traefik:1.3-alpine
Tagging traefik:raclette-alpine
Tagging traefik:alpine

$ test/run.sh traefik:v1.3.8-alpine
testing traefik:v1.3.8-alpine
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...passed

```

**Comment by emilevauge** [comment]
@yosifkit @tianon Just in case you don't already know, it seems the hub is a bit broken https://hub.docker.com/r/containous/traefik/builds.

**Comment by tianon** [comment]
That's distressing! :confused:  That'd be something to email support@docker.com about though (we don't have any access or control on Hub or automated builds). :disappointed: :heart:

**Comment by emilevauge** [comment]
@tianon Sorry, wrong link :blush: I mean that the official has not built: `traefik:v1.4.0-rc3` is still not available at https://hub.docker.com/r/library/traefik/tags. I was thinking that the whole Docker Hub was broken has our image also doesn't build ;)

**Comment by tianon** [comment]
Ah, the official images are now built via Jenkins (since with multiarch, our old mostly-manual process became instantly untenable).  They should kick off fairly soon, but I've triggered them manually now to get them out faster. :+1:

**Pull request merged**